### ISSUE TITLE
PowerShell Core Start-Process cmdlet opening apps DPI Unaware  

### ISSUE THREAD

**Issue by axobel** [PowerShell/PowerShell#13715]
When I open any application using Start-Process in PowerShell Core the **DPI Awareness of the process is Unaware** causing scaling problems with the applications on a HiDPI display. Fonts appear blurry or scaled incorrectly, dialogs are unreadable, etc.. 

I have tested this in a PowerShell Core window and through a Windows Terminal PowerShell Core session and both have the same issue. Windows PowerShell sets the DPI Awareness correctly to System through PowerShell ISE, a PowerShell window, and a Windows Terminal PowerShell session.

The DPI Awareness setting of the running process can be seen through Task Manager > Details (Tab) > DPI Awareness (Column).

## Steps to reproduce

```
1. Run `Start-Process outlook` in any PowerShell Core session.

```

## Expected behavior

```
The application opens with the system default scaling settings (200% on my display panel), with DPI Awareness set to System. 
```

## Actual behavior

```
The application opens with DPI Awareness set to Unaware. 
```

## Environment data

```
Name                           Value
----                           -----
PSVersion                      7.1.0-rc.1
PSEdition                      Core
GitCommitId                    7.1.0-rc.1
OS                             Microsoft Windows 10.0.19041
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0ÔÇª}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0

```
![outlook_dpi_unaware](https://user-images.githubusercontent.com/65114226/94650501-84a11f80-02ff-11eb-8b63-f8a080fa72d2.png)

**Comment by DHowett** [comment]
> Windows PowerShell sets the DPI Awareness correctly to System through PowerShell ISE, a PowerShell window, and a Windows Terminal PowerShell session.

Interesting.

Perhaps there was a change to the .NET process API PowerShell is using between Framework and Core?

The console host cannot control this, so I'm afraid I won't be of much help :smile:

**Comment by iSazonov** [comment]
@axobel Could you reproduce this with a simple C# application?

**Comment by axobel** [comment]
So. I can code in many languages, but I haven't used C# in ages other than using .Net classes in a PowerShell module I am building for my colleagues and I at work. I was going to ask if you knew of an simple open source app I could use to test with but decided to go down another route....

After spending too long searching online for how to set the a programs DPI Awareness property and not coming up with anything concrete in PowerShell, using CIM classes, etc..  I checked the **pwsh.exe DPI Awareness in Task Manager** and it was also Unaware. The problem wasn't occurring before so I think I inadvertently changed it when messing with Soap UI (related to the module that I am building), Java versions and .bat files that don't play well with HiDPI displays. I am moving more and more to working predominantly from the shell in Windows (very good thing) so I was running everything in the shell.

I ran the **Troubleshooter on pwsh.exe and got it to change its DPI Awareness to Per Monitor**. Now, apps started with Start-Process are appearing as they should, so closing the issue. Sorry to waste your time, but thanks a million for the assist.

I would still like to know how to mess with the DPI Awareness or get to the property programmatically though... ¯\_(ツ)_/¯
